stack.yaml

```diff
@@ -1,15 +1,15 @@
-resolver: nightly-2020-03-14
+resolver: nightly-2019-08-07 # nightly-2020-03-14 is giving 404 errors
```

Comment:
Worked fine for me and on CI... Would be good to be on an 8.8.3 release.

Reply:
I don't know what to tell you.
```
shaynefletcher:hlint (update-hlint)$ stack build --resolver nightly-2020-03-14
Downloading nightly-2020-03-14 build plan ...
RedownloadInvalidResponse Request {
  host                 = "raw.githubusercontent.com"
  port                 = 443
  secure               = True
  requestHeaders       = []
  path                 = "/fpco/stackage-nightly/master//nightly-2020-03-14.yaml"
  queryString          = ""
  method               = "GET"
  proxy                = Nothing
  rawBody              = False
  redirectCount        = 10
  responseTimeout      = ResponseTimeoutDefault
  requestVersion       = HTTP/1.1
}
 "/Users/shaynefletcher/.stack/build-plan/nightly-2020-03-14.yaml" (Response {responseStatus = Status {statusCode = 404, statusMessage = "Not Found"}, responseVersion = HTTP/1.1, responseHeaders = [("Connection","keep-alive"),("Content-Length","15"),("Content-Security-Policy","default-src 'none'; style-src 'unsafe-inline'; sandbox"),("Strict-Transport-Security","max-age=31536000"),("X-Content-Type-Options","nosniff"),("X-Frame-Options","deny"),("X-XSS-Protection","1; mode=block"),("Via","1.1 varnish (Varnish/6.0)"),("X-GitHub-Request-Id","8CEC:2E13:DA5F8:11334A:5E6FB36A"),("Accept-Ranges","bytes"),("Date","Mon, 16 Mar 2020 17:12:11 GMT"),("Via","1.1 varnish"),("X-Served-By","cache-lga21971-LGA"),("X-Cache","MISS, MISS"),("X-Cache-Hits","0, 0"),("X-Timer","S1584378731.068852,VS0,VE78"),("Vary","Authorization,Accept-Encoding"),("Access-Control-Allow-Origin","*"),("X-Fastly-Request-ID","7238a14e70621ff2d9051030c039df6fa4d5aa96"),("Expires","Mon, 16 Mar 2020 17:17:11 GMT"),("Source-Age","0")], responseBody = (), responseCookieJar = CJ {expose = []}, responseClose' = ResponseClose})
```

Reply:
What version of Stack are you on? I recommend a stack upgrade. It's possible it no longer looks where it used to.

Reply:
Ah!
```shaynefletcher:hlint (parse-flags)$ stack --version
Version 1.9.3, Git revision 40cf7b37526b86d1676da82167ea8758a854953b (6211 commits) x86_64 hpack-0.31.1```
Let me look into that. Yep,
```
shaynefletcher:hlint (parse-flags *)$ stack --version
Version 2.1.3, Git revision 0fa51b9925decd937e4a993ad90cb686f88fa282 (7739 commits) x86_64 hpack-0.31.2
```
Ok, stand by... Will amend the PR.